config/software/logrotate.rb

```diff
@@ -24,13 +24,8 @@
 
 source url: "https://github.com/logrotate/logrotate/archive/#{version}.tar.gz"
 
-version "3.9.2" do
-  source md5: "584bca013dcceeb23b06b27d6d0342fb"
-end
-version "3.8.5" do
-  source md5: "d3c13e2a963a55c584cfaa83e96b173d",
-         url: "https://fedorahosted.org/releases/l/o/logrotate/logrotate-#{version}.tar.gz"
-end
+version "3.9.2" { source md5: "584bca013dcceeb23b06b27d6d0342fb" }
+version "3.8.9" { source md5: "e6da1f1b91d1f202d26caaf864aa0d71" }
```

Comment:
If you use curly braces like this, then you have to use parens on the arguments. This is getting parsed as a code block "attached to" the preceding String because of precedence.